### ISSUE TITLE
Aria labels for table actions: group and filter

### ISSUE THREAD

**Issue by edenhernandez-suse** [rancher/dashboard#8146]
## Context
Table actions are not properly described. Although they have a description, it is not explicit enough:

![image](https://user-images.githubusercontent.com/120640736/218422725-eaa7a848-9d2f-43e3-8305-cc6dc3fe1edb.png)

This is what VoiceOver says:

![image](https://user-images.githubusercontent.com/120640736/218422788-16f1e910-ed68-4254-961b-5d122c4ef048.png)

This information does not have enough context for users with visual impairments. The same happens with the filter:

![image](https://user-images.githubusercontent.com/120640736/218423298-0aaa9630-b4df-4132-bfb4-699a5098aaf2.png)

## Expected Result
These components should have an aria-label with more context about what they do, e.g:
- Show table without groupings
- Group table by Project
- Filter table content 

Part of #7968

**Comment by aalves08** [comment]
Fixed by https://github.com/rancher/dashboard/pull/13519. Moving this to test

**Comment by a-arias** [comment]
**TESTS**
Rancher : v2.12-7ca04114a589de3edf62e85eded0ba89c408ec2c-head
UI:master https://github.com/rancher/dashboard/commit/28636c14f7ef3a2023074e99ed90cab3300eda53
Browser: chrome, firefox

Fix validated ✅ 

https://github.com/user-attachments/assets/24a3d63d-661a-4de3-ae6a-1818686cae51
